### DISCUSSION TITLE
Platform-specific command line arg for Android-based stores (like Amazon)

### DISCUSSION THREAD

**Discussion by sjabberwocky** [airsdk/apm#140] [General]
Currently `apm` works fine with iOS and Android (Google Play) platforms.
But `apm` does not support Amazon some specific manifest additions.

- Amazon schema - `<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:amazon="http://schemas.amazon.com/apk/res/android" package="air.com.product.id" android:versionCode="1" android:versionName="1.0">`
- Strings for Amazon's inAppPurchases
- Strings for Amazon's One Signal push notifications 
- At the same time it include some related strings for Google Play which does not need for Amazon store

Looks like the same type of improvement will be needed for Huawei platform too (currently our product does not support it).

**Comment by marchbold** [comment]
Can you elaborate? This should work through the custom android manifest additions.

https://github.com/airsdk/apm/wiki/Usage-Generate#android
